components/rating/rating.js

```diff
@@ -0,0 +1,22 @@
+const ratingStar = [...document.getElementsByClassName('rating-star')];
+
+// reference
+// https://dev.to/leonardoschmittk/how-to-make-a-star-rating-with-js-36d3
+ratingStar.map((item) => {
+    starActive = 'rating-star rating-star-checked fas fa-star';
+    starInactive = 'rating-star far fa-star';
+
+    item.addEventListener('click', () => {
+        let i = ratingStar.indexOf(item);
```

Comment:
can use const though

Reply:
the value of variable is changing that is why i used let.